scripts/PearlPowderISIS/pearl_routines.py

```diff
@@ -936,38 +687,18 @@ def PEARL_createvan(van, empty, ext="raw", fmode="all", ttmode="TT88",
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=1)
-        print "About to strip Work=2"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=2)
-        print "About to strip Work=3"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=3)
-        print "About to strip Work=4"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=4)
-        print "About to strip Work=5"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=5)
-        print "About to strip Work=6"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=6)
-        print "About to strip Work=7"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=7)
-        print "About to strip Work=8"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=8)
-        print "About to strip Work=9"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=9)
-        print "About to strip Work=10"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=10)
-        print "About to strip Work=11"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=11)
-        print "About to strip Work=12"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=100, Tolerance=10, WorkspaceIndex=12)
-        print "About to strip Work=13"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=60, Tolerance=10, WorkspaceIndex=13)
+
+        for i in range(1, 12):
+            print "About to strip Work=" + str(i)
+            StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=15, Tolerance=8, WorkspaceIndex=i)
 
         # run twice on low angle as peaks are very broad
-        print "About to strip Work=12 (again)"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=100, Tolerance=10, WorkspaceIndex=12)
-        print "About to strip Work=13 (again)"
-        StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=60, Tolerance=10, WorkspaceIndex=13)
+        print("About to strip work=12 and work=13 twice")
+        for i in range(0, 2):
+            print "About to strip Work=12"
+            StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=100, Tolerance=10, WorkspaceIndex=12)
+            print "About to strip Work=13"
+            StripPeaks(InputWorkspace="vanstrip", OutputWorkspace="vanstrip", FWHM=60, Tolerance=10, WorkspaceIndex=13)
 
```

Comment:
@FedeMPouzols Yes apparently `12` and `13` are meant to be stripped twice; as the comment above says *run twice on low angle as peaks are very broad*